addons/sys_intercom/fnc_enterVehicle.sqf

```diff
@@ -0,0 +1,67 @@
+/*
+ * Author: ACRE2Team
+ * Handles entering a vehicle
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Unit entering a vehicle <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [cursorTarget, player] call acre_sys_intercom_fnc_enterVehicle
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_vehicle", "_unit"];
+
+if (_unit != _vehicle) then {
+    _unit setVariable [QGVAR(intercomVehicle), _vehicle];
+
+    // Configure the array for handling if a player is in a limited intercom position or not.
+    private _usingLimitedPosition = [];
+    {
+        _usingLimitedPosition pushBack false;
+    } forEach (_vehicle getVariable [QGVAR(intercomNames), []]);
```

Comment:
24-28 can be replaced with:
`private _usingLimitedPosition = (_vehicle getVariable [QGVAR(intercomNames), []]) apply {false};`